Polygon_mesh_processing/examples/Polygon_mesh_processing/triangulate_faces_split_visitor_example.cpp

```diff
@@ -0,0 +1,105 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
```

Comment:
You have to add it to doc/*/examples.txt

Can you also add it in the User Manual